File: src/features/pix/redux/models/response/PixAddress.ts
import { AddressType } from "./enum";

export interface PixAddress{
    addressLine1: string;
    addressLine2: string;
    zipCode: string;
    neighborhood: string;
    cityName: string;
    state: string;
    addressType: AddressType;
    country: string;
    complement: string;
}

File: osb-core-main/Platform/Integration/Osb.Core.Platform.Integration.Entity/FitBank/Models/PixAddress.cs
using Osb.Core.Platform.Common.Entity.Enums;

namespace Osb.Core.Platform.Integration.Entity.FitBank.Models
{
  public class PixAddress
  {
    public string AddressLine1 { get; set; }
    public string AddressLine2 { get; set; }
    public string ZipCode { get; set; }
    public string Neighborhood { get; set; }
    public string CityName { get; set; }
    public string State { get; set; }
    public AddressType AddressType { get; set; }
    public string Country { get; set; }
    public string Complement { get; set; }
  }
}